### PR TITLE
change html and css file

### PR DESCRIPTION
**Pull request by masouma2020**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -20,66 +20,80 @@
               <img src="images/Capacity.png" alt="Capacity" />
             </div>
 
-            <img src="images/Time.png" alt="time" id="logo" />
+            <img src="images/Time.png" alt="time" id="time" />
           </div>
           <div id="toolbar">
```
Comment (MeqdamAlqudah) [inline index.html:25]: - [ ]  Please as the last reviewer said you don't need to implement this part of the design 
image]

- [Optional ] Please provide a good description for this pull request

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @masouma2020 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
:+1: Readme file

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by smunozmo** [comment]
Hi , 👋  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

# Invalid Code Review Request ❌ 

- You have submitted a Pull Request that:

- [ ]  Has no PR description, please add some information about this PR in the description as this is a general requirement. 
- [ ] Has no information about the project in the README file.
- [ ] No changes has been made since the last review, please address the suggested changes and submit it again for review.


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

⚠️  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html (hunk 1 above)
Comment (masouma2020) [reply index.html:25]: ok no matter.
now that i did it.

**Comment by masouma2020** [comment]
Please approve it 
The time is over.

**Review by mgmediaweb — CHANGES_REQUESTED** [review]
Hi  @masouma2020, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️

*** Kindly, I recommend you to create more descriptive Pull Request

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by smunozmo** [comment]
> Please approve it The time is over.

Hi @masouma2020,
please try to implement the suggested changes before submitting it again for review, otherwise, the review would be invalid.

You are almost there!

**Pull request merged**